cloudos/jobs/job.py

```diff
@@ -385,8 +388,9 @@ def convert_nextflow_to_json(self,
             "resumable": resumable,
             "saveProcessLogs": save_logs,
             "batch": {
-                "dockerLogin": False,
+                "dockerLogin": docker_login,
                 "enabled": batch,
+                "id": job_queue_id,
```

Comment:
I don't think we need to pass the job queue id as id field here

Reply:
@s0rthak I've removed that field and tested again with identical results than the ones in the PR description. So you were right, that `id` field was not required. Removed :+1: Thanks!